pkl-core/src/main/java/org/pkl/core/runtime/VmMapping.java

```diff
@@ -113,21 +117,6 @@ public Map<Object, Object> export() {
     return properties;
   }
 
-  @TruffleBoundary
-  public Map<Object, Object> toMap() {
-    var properties = CollectionUtils.newLinkedHashMap(EconomicMaps.size(cachedValues));
-
-    forceAndIterateMemberValues(
-        (key, prop, value) -> {
-          if (isDefaultProperty(key)) return true;
-
-          properties.put(key, value);
-          return true;
-        });
-
-    return properties;
-  }
```

Comment:
This method is not being used, _and_ it uses the size of `cachedValues`, which is not kosher. So, just get rid of it.